apple/internal/apple_framework_import.bzl

```diff
@@ -119,6 +131,21 @@ def _apple_framework_import_impl(ctx):
             objc_provider_fields["weak_sdk_framework"] = depset(ctx.attr.weak_sdk_frameworks)
         objc_provider_fields["static_framework_file"] = framework_imports_set
 
+        bundle_files = [x for x in framework_imports if ".bundle/" in x.short_path]
```

Comment:
Is there a more efficient solution than this? It seems like most of the functions in the `if` below require at least some files to contain `.bundle` so we can't call them without filtering first since this is optional